deepbox/src/main/java/ch/cyberduck/core/deepbox/DeepboxListService.java

```diff
@@ -285,6 +323,40 @@ public AttributedList<Path> list(final Path directory, final ListProgressListene
                     offset += chunksize;
                 }
                 while(offset < size);
+                final Path shared = new Path(directory, containerService.getPinnedLocalization(SHARED), EnumSet.of(Path.Type.directory, Path.Type.volume));
+                if(!new SharedWithMeListService(companyId).list(shared, listener).isEmpty()) {
```

Comment:
Is it a requirement that the _Shard with me_ folder is only displayed conditionally?

Reply:
Yes.